lib/browser.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function (self, options) {
+
+  self.pushAsset('script', 'always', { when: 'always' });
+
+  self.pageBeforeSend = function (req) {
+    self.pushCreateSingleton(req, 'always');
+  };
+
+  self.getCreateSingletonOptions = function (req) {
+    return {
+      currentExp: req.experience
```

Comment:
Experiences differ from personas in that a customer may have lorries, flurries and scurries in their fleet at the same time. So there is no current experience, there is a current array of experiences. Which gets updated when they log in via a salesforce query that is implemented per project.